README.md

```diff
@@ -104,37 +104,27 @@ Filesystem event monitoring: The provided filesystem layer implements
 the Linux `inotify` and the MacOS `fsevents` interfaces.
 
 To build the reference `megacli` example, you may also need to install:
-
 * GNU Readline (`libreadline-dev`, `readline-devel`)
+on Mac, you will probably need to download the source and build it yourself, and adjust the project to refer to that version.
 
 For Android, we provide an additional implementation of the graphics subsystem using Android libraries.
 
 For iOS, we provide an additional implementation of the graphics subsystem using Objective C frameworks.
 
 #### Windows
 
-To build the client access engine under Windows, you'll need the following:
-
-* A Windows-native C++ development environment (e.g. MinGW or Visual Studio)
-* Crypto++
-* zlib (until WinHTTP learns how to deal with Content-Encoding: gzip)
-* SQLite or configure `--without-sqlite`
-* FreeImage or configure `--without-freeimage`
-* pthreads (MinGW)
-
-Optional dependency:
-* Sodium, configure `--with-sodium`
+We use vcpkg, cmake, and Visual Studio, and provide scripts to build the 3rd party libraries and set up the project.
 
 ### Build Dependencies
-* Consult contrib\cmake\3rdparty_deps.txt to enable/disable a particular package.
+* Consult contrib\cmake\3rdparty_deps.txt to see dependencies and enable/disable a particular package.
 * Your 3rdParty library builds should be outside the SDK repo. You can use the same 3rdParty folder for other mega components, e.g., megasync, etc..
 * We are moving to use vcpkg to build most of them. You can start like this:
```

Comment:
We are no longer moving for Windows ;-) Also this title should be a subsection within Windows section?

Reply:
Yes, good point.   Adjusted, and also rearranged the sections so that vcpkg+cmake is first (with notes on windows/posix differences), with the posix autotools section after.